### DISCUSSION TITLE
Email's template - Create URL

### DISCUSSION THREAD

**Discussion by frato92** [YetiForceCompany/YetiForceCRM#14425] [How to]
Hi, I have a problem to create the email's template. When I create the link at my website, this link is create incorrectly because there isn't the hipertextual link on the website.I checked the source and before the saving, the tag contains the link, but when i proceed to save it, the link isn't in the tag.

The developer tell's me to set domains in configuration. How i can do this?

Thanks for the support.

**Comment by mariuszkrzaczkowski** [comment]
https://github.com/YetiForceCompany/YetiForceCRM/issues/14263

**Reply by frato92** [reply]
How Set domains in configuration ?

**Reply by mariuszkrzaczkowski** [reply]
edit the file
